freva-data-portal-worker/src/data_portal_worker/load_data.py

```diff
@@ -247,9 +243,12 @@ def load_object(self, key: str) -> LoadDict:
         KeyError: If the key doesn't exist in the cache (anymore).
         """
         data_cache = self.cache.get(key)
```

Comment:
i thinnk we have to keep a better naming convention here to make it more understandable for first time developer who jump to fix the bug. like, `dset_cache` and `meta_cache`

Reply:
Ok